infra/README.md

```diff
@@ -23,31 +23,29 @@ The resources/services/activations/deletions that this module will create/trigge
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 ## Inputs
+
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| bucket\_location | Bucket location. https://cloud.google.com/storage/docs/locations | `string` | `"US"` | no |
-| data_init_bucket_name | Name of the Cloud Storage bucket that store the archive file for initialization. | `string` | jss-resources | no |
-| data_init_archive_file_name | Name of the archive file that contains the initialization data. | `string` | initialization.tar.gz | no |
-| disable\_services\_on\_destroy | Whether project services will be disabled when the resources are destroyed. | `bool` | `false` | no |
-| init | Initialize resource or not | `bool` | `true` | no |
-| labels | A map of key/value label pairs to assign to the resources. | `map(string)` | <pre>{<br>  "app": "large-data-sharing-java"<br>}</pre> | no |
-| lds\_client\_image | Docker image for frontend | `string` | `"gcr.io/hsa-resources-public/hsa-lds-java-frontend:latest"` | no |
-| lds\_initialization\_archive\_file\_name | Archive file's name in lds-initialization bucket | `string` | `"initialization.tar.gz"` | no |
-| lds\_initialization\_bucket\_name | Bucket for cloud run job | `string` | `"jss-resources"` | no |
-| lds\_server\_image | Docker image for backend | `string` | `"gcr.io/hsa-resources-public/hsa-lds-java-backend:latest"` | no |
+| bucket\_location | Bucket location where the uploaded files will be stored. | `string` | `"US"` | no |
+| data\_init\_archive\_file\_name | Name of the archive file that contains the initialization data. | `string` | `"initialization.tar.gz"` | no |
+| data\_init\_bucket\_name | Name of the Cloud Storage bucket that store the archive file for initialization. | `string` | `"jss-resources"` | no |
+| labels | A map of key/value label pairs to assign to the cloud resources. | `map(string)` | <pre>{<br>  "app": "cloud-deployment-gce-java"<br>}</pre> | no |
 | project\_id | GCP project ID. | `string` | n/a | yes |
-| region | Google cloud region where the resource will be created. | `string` | `"us-central1"` | no |
-| source_image | Source VM image to deploy on GCE. | `string` | `"https://www.googleapis.com/compute/beta/projects/hsa-public/global/images/jss-cd-gce-vm-image"` | no |
-| zones | Gogle cloud zone where resources will be created. | `string` | `us-central1-a` | no |
+| region | GCP region where the cloud resource will be created. | `string` | `"us-central1"` | no |
```

Comment:
We're not supposed to use "GCP", the term has been deprecated in favour of "Google Cloud"

Reply:
Same comment elsewhere in this README.